plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/changedList/ChangedListViewImpl.java

```diff
@@ -54,21 +62,28 @@
     private static ChangedListViewImplUiBinder ourUiBinder = GWT.create(ChangedListViewImplUiBinder.class);
 
     @UiField
-    DockLayoutPanel changedFilesPanel;
+    LayoutPanel changedFilesPanel;
```

Comment:
Your ui.xml uses `DockLayoutPanel`, here you use `LayoutPanel`. Use one type.
I'd recommend you to use `FlowPanel`, it has less problems with dynamically positioning.